android/build.gradle

```diff
@@ -5,33 +5,36 @@ buildscript {
         buildToolsVersion = "30.0.2"
         minSdkVersion = 21
         compileSdkVersion = 30
-        targetSdkVersion = 29
-        ndkVersion = "20.1.5948944"
+        targetSdkVersion = 30
+        ndkVersion = "21.4.7075529"
     }
     repositories {
         google()
         jcenter()
```

Comment:
I think we should remove jcenter it has been deprecated

Reply:
The guide keeps it there, I think I'll leave it, but move it below mavenCentral. According to jcenter they are going to keep it available as read only.